README.md

```diff
@@ -144,19 +155,50 @@ stop and restart the service. Use these if you want to try different settings
 in your installed `config.yml` or other YML files (such as `filter.yml`).
 
 
-### On Windows
+### On Windows (Command Prompt)
+
+To build a ZIP file for Windows using a Developer Command Prompt and create
```

Comment:
"Developer Command Prompt" here means the Visual Studio Developer Command Prompt, and not a 'regular' Windows cmd.exe prompt window, right? May be worth calling this out explicitly.. that it requires the **VS** specific setup.

Reply:
Good point.  TBH I haven't tried it in a plain cmd prompt.  Hey, it works with a plain one.  I clarify it.  Thanks!